src/Components/ServiceScene/ActionBarScene.tsx

```diff
@@ -39,6 +41,10 @@ export class ActionBarScene extends SceneObjectBase<ActionBarSceneState> {
       <Box paddingY={0}>
         <div className={styles.actions}>
           <Stack gap={1}>
+            {
+              // @ts-ignore appSidecar not yet in stable runtime
```

Comment:
```suggestion
              // @ts-expect-error appSidecar not yet in stable runtime
```

Reply:
Hmm actually I think the `@ts-expect-error` is wrong here (well also IDE says so :D). I assume the thing is the `config.featureToggles.appSidecar` does not error out in a regular sense, the `appSidecar` will just be `undefined`.
<img width="394" alt="Screenshot 2024-10-30 at 16 36 33" src="https://github.com/user-attachments/assets/0ca5d283-6e71-47ad-8847-b2c815488fa6">

Reply:
When we get that it usually means that the feature flag has been released and is present in the type. In any case, it's pretty minor. Thanks for checking!